frontend/src/views/TripList.vue

```diff
@@ -21,6 +21,7 @@
         <h4>New Trip</h4>
         <hr/>
         <input type="text" v-model="newTrip.name" placeholder="Trip name (ex. The best trip ever!)"></input>
+        <input type="text" v-model="newTrip.picture" placeholder="Photo URL (ex. photos.com/myphoto.jpg)"></input>
```

Comment:
is "photos.com/myphoto.jpg" okay here? we don't need 'www' or 'http'?

Reply:
Uh let me check.

Reply:
Looks like we need that. Thanks for the catch.